# HG changeset patch
# User Rik <rik@octave.org>
# Date 1384280579 28800
#      Tue Nov 12 10:22:59 2013 -0800
# Node ID c9b3951a9d6b0707aa7eaedea3a778d3d535210c
# Parent  817da22e58edef23326abaa7bba09e823ae840b2
octave.texi: Update copyright on manual to include 2013.

* octave.texi: Update copyright on manual to include 2013.

diff --git a/doc/interpreter/octave.texi b/doc/interpreter/octave.texi
--- a/doc/interpreter/octave.texi
+++ b/doc/interpreter/octave.texi
@@ -104,17 +104,17 @@ modified versions.
 @end multitable 
 @author John W. Eaton
 @author David Bateman
 @author S@o{}ren Hauberg
 @author Rik Wehbring
 @page
 @vskip 0pt plus 1filll
 Copyright @copyright{} 1996, 1997, 1999, 2000, 2001, 2002, 2005, 2006,
-2007, 2011 John W. Eaton.
+2007, 2011, 2013 John W. Eaton.
 
 This is the third edition of the Octave documentation, and is consistent
 with version @value{VERSION} of Octave.
 
 Permission is granted to make and distribute verbatim copies of
 this manual provided the copyright notice and this permission notice
 are preserved on all copies.
 
